client/src/modules/shopping-list/model/reducer.js

```diff
@@ -28,14 +28,21 @@ const shoppingLists = (state = {}, action) => {
         [action.payload.listId]: updatedList
       };
     }
-    case ShoppingListActionTypes.FETCH_PRODUCTS_SUCCESS: {
-      const updatedShoppingList = {
-        ...state[action.payload.listId],
-        products: action.payload.products
+    case ShoppingListActionTypes.ARCHIVE_SUCCESS: {
+      const archivedList = {
+        _id: action.payload.listId,
```

Comment:
is this _id property needed to update??
I can see that you are updating using the key property `[action.payload.listId]` so maybe try update without this _id property

Reply:
we override full list data object with new data with id and isArchive